Makefile

```diff
@@ -148,3 +148,10 @@ swagger-gen:
 		--entrypoint hack/generate-swagger-api.sh \
 		-e GOPATH=/go \
 		quay.io/goswagger/swagger:0.7.4
+
+.PHONY: swagger-docs
+swagger-docs:
+	@docker run --rm -v $(PWD)/api/swagger.yaml:/usr/share/nginx/html/swagger.yaml \
+		-e 'REDOC_OPTIONS=hide-hostname="true" suppress-warnings="true" lazy-rendering' \
+		-p 80:80 \
```

Comment:
Not sure this is great.
Maybe we can move this into a `hack/make/swagger-docs` script that can just do a `-p` and then report the port being exposed.
Or an env var like `DOCKER_SWAGGER_PORT`.
But still set the host port to something like 3000 instead of 80.